controllers/favorites.js

```diff
@@ -0,0 +1,32 @@
+var express = require("express");
+var router = express.Router();
+var db = require('../models');
+// current directory : localhost:3000/favorites
+
+
+/// localhost:3000/favorites/
+router.get("/", function(req, res) {
+	db.favorite.findAll().then(function(fav) {
+		res.render('favorites.ejs', {fav:fav})
+	})
+	//this is where I will list the database entries
+});
+
+router.post("/", function(req, res) {
+	var title = req.body.favtitle;
+	var year = req.body.favyear;
+	var imdb = req.body.favimdb;
+
+	db.favorite.create({
+		imdbId: imdb,
+		title: title,
+		year: year
+	});
```

Comment:
Let's put the res.redirect into a promise here. Also, I'd advise that you redirect the user to favorites, so that they can see their recently favorited movie.

```
	}).then(function(created){
		console.log('created movie')
		res.redirect("/favorites");
	});
```